.github/workflows/linux.yml

```diff
@@ -321,6 +310,15 @@ jobs:
       BUILD_DIR: /__w/openvino/openvino/build
 
     steps:
+
+      - name: Fetch setup_python action
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            .github/actions/setup_python/action.yml
+          sparse-checkout-cone-mode: false
+          path: 'openvino'
```

Comment:
I think it is not important to checkout actions to the same folder, let's use independent directory, for example "actions"

Reply:
or without path at all, it will reduce the code

Reply:
Sparse-checkout works as a normal git clone so that the paths for all files (`.github/actions/setup_python/action.yml` in this case) are preserved and the full path would still be the same whether we use `openvino` as a path or do not use a path at all. I suggest keeping it like this for consistency with other checkouts and the fact that it is the `openvino` repository.